### ISSUE TITLE
KeepassXC crash on Ubuntu when opening a file from google drive

### ISSUE THREAD

**Issue by afxgroup** [keepassxreboot/keepassxc#10643]
## Overview
KeepassXC crash on Ubuntu when opening a file from google drive

## Steps to Reproduce
1. Try to open on Ubuntu a file stored in google drive. Once you try to open it the program hangs

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - Versione 2.7.6
Revisione: dd21def
Distribuzione: Snap

Qt 5.15.9
La modalità debug è disabilitata.

Sistema operativo: Ubuntu Core 22
Architettura CPU: x86_64
Kernel: linux 6.5.0-28-generic

Estensioni abilitate:
- Completamento automatico
- Integrazione browser
- Agente SSH
- KeeShare
- YubiKey
- Integrazione Secret Service

Librerie crittografiche:
- Botan 2.19.1

**Comment by droidmonkey** [comment]
Does the file open fine if you copy it to your documents folder or similar? 

Also recommend switching to the flatpak distribution, snaps are terrible.

**Comment by afxgroup** [comment]
Yes, copying the same file on a regular folder works without any problems

**Comment by droidmonkey** [comment]
Then this isn't a keepassxc crash. It's likely a fault in the snap interface.

**Comment by afxgroup** [comment]
I've installed the flatpak one and there is the same problem. So it isn't a snap interface problem

```KeePassXC - Versione 2.7.7
Revisione: 68e2dd8
Distribuzione: Flatpak

Qt 5.15.10
La modalità debug è disabilitata.

Sistema operativo: KDE Flatpak runtime
Architettura CPU: x86_64
Kernel: linux 6.5.0-28-generic

Estensioni abilitate:
- Completamento automatico
- Integrazione browser
- Passkey
- Agente SSH
- KeeShare
- YubiKey
- Integrazione del servizio segreto

Librerie crittografia:
- Botan 3.3.0
```

**Comment by droidmonkey** [comment]
Will take a look, but this still doesn't seem like a crash on our code, per say. We don't really care where a file exists, as long as qt can handle it for us.

**Comment by martimors** [comment]
Did you ever get this working @afxgroup ? Would be neat to use keepassxc in this way, but seems to crash on Ubuntu 24.04.

Edit: Switched to using syncthing, couldn't be happier to de-google.

**Comment by droidmonkey** [comment]
Not reproducible
